padglobal/view/view_state_base.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
```

Comment:
seems kinda silly to say this because we will prob need it in the future but currently I don't think we need this file; we can just use the base view state from discord-menu. so can you delete (for now at least)? mostly because this will make this PR an easier reference example for people learning to understand discordmenu lol